frontend/src/features/mapSideBar/MotiInventoryHeader.tsx

```diff
@@ -57,7 +57,6 @@ export const MapSlideBarHeader: React.FunctionComponent<IMapSlideBarHeaderProps>
 };
 
 const HeaderWrapper = styled.div`
-  position: relative;
```

Comment:
Ah, I see, another display regression then, likely caused by this. without an absolute/relative parent the spinner doesn't know what to draw on top of and so is not drawing on top of the header correctly anymore.

Reply:
Now there is just one spinner that takes up the whole screen, I would consider that a regression as well.